apps/store/modules/asset-api.js

```diff
@@ -540,4 +540,24 @@ var responseProcessor = require('utils').response;
         paging.url = url;
         return paging;
     };
+    
+    api.formatSearchQuery = function (queryString) {
+        var searchQuery = "";
+        var qjson = parse('{' + queryString + '}');
+
+        var searchKeys = Object.keys(qjson);
+        if ((searchKeys.length === 1) && (searchKeys.indexOf("name") >= 0)) {
+            searchQuery += qjson[searchKeys.pop()];
+        }
+        else {
+            for (var keyIndex in searchKeys) {
+                var key = searchKeys[keyIndex];
+                var value = qjson[key];
+                searchQuery += key + ":" + value + " ";
+            }
+        }
+        searchQuery = searchQuery.trim();
+        return searchQuery;
+    };
+
```

Comment:
Even though there is no difference between using single quotes over double quotes, try to maintain the consistency throughout the code.  L546 you have used ''